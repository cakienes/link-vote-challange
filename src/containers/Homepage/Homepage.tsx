import React from 'react';
import Layout from '../Layout/Layout';
import LinkList from './LinkList/LinkList';
import SubmitNewLink from './SubmitNewLink/SubmitNewLink';

const Homepage: React.FC = () => {
    return (
        <Layout>
            <SubmitNewLink />
            <LinkList />
        </Layout>
    );
};

export default Homepage;
