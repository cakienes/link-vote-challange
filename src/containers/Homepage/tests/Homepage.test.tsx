import { shallow } from 'enzyme';
import React from 'react';
import Homepage from '../Homepage';

describe('Homepage.tsx', () => {
    it('renders without crashing', async () => {
        expect(shallow(<Homepage />)).toMatchSnapshot();
    });
});
