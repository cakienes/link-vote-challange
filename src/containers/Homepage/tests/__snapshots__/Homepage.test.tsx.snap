// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage.tsx renders without crashing 1`] = `ShallowWrapper {}`;
